src/ape/managers/chain.py

```diff
@@ -1196,6 +1202,37 @@ def instance_at(
             else:
                 raise  # Current exception
 
+        if abi:
+            # if abi is type string then convert it to json object
+            if isinstance(abi, str):
+                try:
+                    list_abi = json.loads(abi)
+                    contract_type = ContractType(abi=list_abi)
```

Comment:
Would move this line outside of the try context

Also would handle if it's not a list